### PR TITLE
TST: OmniSciDB enable literal_multi_geospatial_inferred test

### PR DESCRIPTION
**Pull request by semelianova**

### CODE REVIEW

**Comment by anmyachev** [comment]
LGTM. @xmnlab what do you think about?

File: ci/docker-compose.yml [hunk 1]
```diff
@@ -89,7 +89,7 @@ services:
       KUDU_MASTER: "false"
 
   omniscidb:
-    image: omnisci/core-os-cpu:v5.1.0
+    image: omnisci/core-os-cpu-dev:latest
```
Comment (xmnlab) [inline ci/docker-compose.yml]: @semelianova does it really need change omniscidb image here?
if yes, we need to wait for #2084

File: ibis/tests/all/test_geospatial.py [hunk 2]
```diff
@@ -205,13 +205,23 @@ def test_literal_geospatial_inferred(backend, con, shp, expected):
             {
                 'postgres': (
                     "'MULTILINESTRING ((0 0, 1 1, 2 2), (2 2, 1 1, 0 0))'"
-                )
+                ),
```
Comment (xmnlab) [inline ibis/tests/all/test_geospatial.py]: thanks for adding these tests!

**Comment by jreback** [comment]
thanks @semelianova

**Pull request merged**